frontend/src/v4/routes/components/dialogContainer/components/errorDialog/errorDialog.styles.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+export const Container = styled.div``;
```

Comment:
hmm, I'm not sure how I feel about these unstyled divs. Would it perhaps be better to use ids instead? That way you don't need a style file for each dialog and you can still add the styles in the overrides. Wait until @sanmont3drepo give his feedback first